data-raw/variable_description.csv

```diff
@@ -0,0 +1,40 @@
+"","register_name","raw_register_filename","years_covered","variable_name","danish_description","english_description","data_type","format","expected_values","explanation_of_values","notes_for_devs"
```

Comment:
I think we should have two columns for the years: `start_year` and `end_year`.

Reply:
Implemented